### PR TITLE
Rephrasing sentence to make it clearer after customer request

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-806.surge.sh

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 1]
```diff
@@ -137,7 +137,8 @@ The default is `false`, to read the claim from the token.
 
 First, you configure Neo4j to use OpenID Connect as an authentication and authorization provider.
 
-. Uncomment the setting `dbms.security.auth_enabled=false` and change its value to `true` to enable the security feature.
+. Make sure security is turned on. 
+The `dbms.security.auth_enabled` setting is `true` by default.
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:141]:
```suggestion
The default value for `dbms.security.auth_enabled` is `true`.
```

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @lidiazuin - one small editorial suggestion.

I did notice a couple of other things in this page that may need a look at, but not within the scope of this PR:

- I can't find anywhere in the instructions that tell the user that these settings are configured in the neo4j.conf file.
- the bullets in [OpenID Connect Using JWT Claims](https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-configure-provider-jwt) have some inconsistent grammar - missing full stops, and occasionally some sentences which follow on from each other, but both have colons, such as:
> Configure Audience:
> Provide the expected value for the audience(aud) claim:

**Comment by lidiazuin** [comment]
> Thanks @lidiazuin - one small editorial suggestion.
> 
> I did notice a couple of other things in this page that may need a look at, but not within the scope of this PR:
> 
> * I can't find anywhere in the instructions that tell the user that these settings are configured in the neo4j.conf file.
> * the bullets in [OpenID Connect Using JWT Claims](https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-configure-provider-jwt) have some inconsistent grammar - missing full stops, and occasionally some sentences which follow on from each other, but both have colons, such as:
> 
> > Configure Audience:
> > Provide the expected value for the audience(aud) claim:

Good catch, David! I committed a few changes now to improve these inconsistencies. Would you mind taking another look?

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 2]
```diff
@@ -197,7 +198,7 @@ It is recommended to use a name describing the provider that is being integrated
 
 In this configuration, Neo4j receives a JWT from the identity provider containing claims representing the database username (e.g. email), and the Neo4j roles.
 
-. Set a Display Name
+. *Set a Display Name*
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:201]:
```suggestion
. Set a display name.
```

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 3]
```diff
@@ -207,7 +208,7 @@ dbms.security.oidc.mysso.display_name=SSO Provider
 ----
 This is displayed on a button on the login page of clients such as Neo4j Browser and Bloom, so that users can identify the provider they are using to login.
 
-. Configure Discovery:
+. *Configure Discovery*
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:211]:
```suggestion
. Configure discovery.
```

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 4]
```diff
@@ -228,7 +229,7 @@ dbms.security.oidc.mysso.issuer=abcd1234
 ----
 Manual settings always take priority over those retrieved from the discovery endpoint.
 
-. Configure Audience:
+. *Configure Audience*
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:232]:
```suggestion
. Configure audience.
```

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 5]
```diff
@@ -244,30 +245,31 @@ In this situation, the id_token should contain an authorized party(`azp`) claim
 dbms.security.oidc.mysso.claims.client_id=myclientid
 ----
 
-. Configure Claims:
+. *Configure Claims*
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:248]:
```suggestion
. Configure claims.
```

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 6]
```diff
@@ -244,30 +245,31 @@ In this situation, the id_token should contain an authorized party(`azp`) claim
...
 
 [[auth-sso-configure-provider-userinfo]]
 === OpenID Connect Fetching Claims from Provider
 
 In this configuration, Neo4j receives a token from the identity provider and uses that token to call back to the identity provider using its UserInfo endpoint to retrieve claims for the database username and Neo4j roles.
 
-. Configure as for JWT Claims.
+. *Configure as for JWT Claims*
 +
 Configure Neo4j for xref:authentication-authorization/sso-integration.adoc#auth-sso-configure-provider-jwt[OpenID Connect Using JWT Claims].
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:270]: I think this looks a little repetitive, and can be simplified as a one-liner:

```suggestion
. Configure Neo4j for xref:authentication-authorization/sso-integration.adoc#auth-sso-configure-provider-jwt[OpenID Connect Using JWT Claims].
```

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 7]
```diff
@@ -244,30 +245,31 @@ In this situation, the id_token should contain an authorized party(`azp`) claim
...
 
-. Configure the claims to fetch from the UserInfo endpoint:
+. *Configure the claims to fetch from the UserInfo endpoint*
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:272]:
```suggestion
. Configure the claims to fetch from the _UserInfo_ endpoint:
```

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 8]
```diff
@@ -244,30 +245,31 @@ In this situation, the id_token should contain an authorized party(`azp`) claim
...
 
-. Optionally, map the groups in the OIDC groups claim to the Neo4j built-in and custom roles.
+. *Optionally, map the groups in the OIDC groups claim to the Neo4j built-in and custom roles*
++
 See xref:authentication-authorization/sso-integration.adoc#auth-sso-map-idp-roles[Map the Identity Provider Groups to the Neo4j Roles].
 
 [[auth-sso-configure-provider-userinfo]]
 === OpenID Connect Fetching Claims from Provider
 
 In this configuration, Neo4j receives a token from the identity provider and uses that token to call back to the identity provider using its UserInfo endpoint to retrieve claims for the database username and Neo4j roles.
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:266]:
```suggestion
In this configuration, Neo4j receives a token from the identity provider and uses that token to call back to the identity provider using its _UserInfo_ endpoint to retrieve claims for the database username and Neo4j roles.
```

**Review by davidoliverSP2 — COMMENTED** [review]
Thanks @lidiazuin 

The bold text in the steps looked a little odd and inconsistent (plus used camel-casing). I think by having the first lines as complete sentences is a little easier for users to read. Take a look at my suggestions and see if you think they're improvements.

**Review by lidiazuin — COMMENTED** [review]
Thanks for your review David!

**Pull request merged**